cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -139,10 +145,20 @@ void Run(const std::vector<string>& bag_filenames) {
     check_insert(kOdometryTopic);
   }
 
+  ::ros::Publisher tf_publisher =
+      node.node_handle()->advertise<tf2_msgs::TFMessage>(
+          kTfTopic, kLatestOnlyPublisherQueueSize);
+
+  ::tf2_ros::StaticTransformBroadcaster static_tf_broadcaster;
+
   ::ros::Publisher clock_publisher =
       node.node_handle()->advertise<rosgraph_msgs::Clock>(
           kClockTopic, kLatestOnlyPublisherQueueSize);
 
+  if (urdf_transforms.size() > 0) {
+    static_tf_broadcaster.sendTransform(urdf_transforms);
```

Comment:
Do you mean we can use a regular tf_broadcaster here? There is already one in the class `Node`, but we can't access it from here because it's a private member.